src/AMQP/EventSourcing/eventQueue.ts

```diff
@@ -32,16 +31,22 @@ export async function fetchNext(client: Client, eventType: string) {
 	});
 }
 
-export async function bindMore(client: Client, eventTypes: string[], onEvent: (event: IEvent) => Promise<void>) {
+export async function bindMore<TPayload extends IEventPayload>(
```

Comment:
Tady v tomhle commitu je spoustu změn, které se toho vůbec netýkají... asi commituj jen ty řádky, které se toho týkají... zbytek vrať nebo commituj odděleně. Nicméně tady to je jen asi přeformátování, ale bylo správně.